packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -343,12 +344,15 @@ function FlowsheetDataGrid(props) {
         onRowSelect({ rowId: selectionDetails.rowId, sectionId: selectionDetails.sectionId, isMetaPressed: selectionDetails.isMetaPressed });
       }
     } else if (selectionDetails.isShiftPressed && anchorCell.current !== null) {
-      selectCellRange(selectionDetails.rowId, selectionDetails.columnId, selectionDetails.sectionId, selectionDetails.subsectionId);
+      selectCellRange(selectionDetails.rowId, selectionDetails.columnId, selectionDetails.columnIndex, selectionDetails.sectionId, selectionDetails.subsectionId);
```

Comment:
Shouldn't we be using the column span index instead of the column index?  Consumers would not be set up to drive off a column index.  They are using the column id to find columns.